M2/Macaulay2/d/expr.d

```diff
@@ -295,6 +295,7 @@ export QQClass := newnumbertype();
 export ringFamilyClass := newtypeof(typeClass);
 export inexactNumberTypeClass := newtypeof(ringFamilyClass);
        newbignumbertype():HashTable := newHashTable(inexactNumberTypeClass,inexactNumberClass);
+export RRiClass := newbignumbertype();
```

Comment:
To fix the assertion error, simply move this new line above the line below that says "-- all new types, dictionaries, and classes go just above this line, if possible, so hash codes don't change gratuitously!
"

Reply:
Thanks - I made the change.  Obviously, I missed the comment about where to add new types.